docs/leap_sapi/index.rst

```diff
@@ -157,45 +116,33 @@ time\ [#]_ for that project is sufficient.
 .. [#] A user's solver-access time for a project is renewed monthly, subject to
     the customer contract; the renewal date is displayed on the dashboard.
 
-.. _leap_support_for_dev_env:
-
-Support for IDEs
-================
-
```

Comment:
why cut? consider retaining this

Reply:
Because anyone coming to this page looking for information on IDE support will find it here (the top-left item is the most visible. Reducing content to the minimum was the most crucial lessons of the usability testing):

![image](https://github.com/user-attachments/assets/adc6357e-de9a-457e-b471-62e8e53cd2d1)

Having a summary of that content in the body will likely be more of a distraction for users looking for other information than an assistance to users looking for IDE support who might have missed the link to the dedicated page.